keyboards/satan/keymaps/admiralStrokers/keymap.c

```diff
@@ -0,0 +1,194 @@
+#include "satan.h"
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
This line should probably be moved down to where your keymap starts - I don't think this is messing anything up though.